apostrophecms/apostrophe
# Pro 2223 batch operations modal

[PRO-2223](https://linear.app/apostrophecms/issue/PRO-2223/as-an-editor-i-can-confirm-starting-a-batch-operation-after-clicking)

## Purpose
Opening confirmation modal when clicking on batch operation button, uses the informations from the `modalOptions` object of the batch operation. Also works with grouped operations. Count and label are passed to all translation keys passed from the batch operations config, since this is the only dynamic data needed.

## How to test
Open a piece module modal, click on the archive button, it should open a modal with the text passed from the batch operations config. You can test with grouped ones, to do that add `archive` to the group config.
